CustomeDB/CustomeDB.cpp

```diff
@@ -94,3 +94,96 @@ int main()
 
     return 0;
 }
+/////////////////////////////////////////////////////////////////////
+//Hamza was here L
+#include <iostream>
+#include <string>
+using namespace std;
+
+struct Node {
```

Comment:
there is memory leakage in your program. i want u to find it out. for hint run add data and delete at head hundreds of thousands of time try testing it.